modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -19,97 +17,43 @@ When viewing or modifying a graph, Anypoint Visualizer provides the following fu
 
 * Zoom control: Use the zoom in, zoom out, and zoom fit buttons on the bottom right corner of the canvas to control the zoom. You can also use scroll to zoom in and out.
 * Pan the visualization: Click and drag on the canvas to pan the visualization.
+* Search: Search for the name of a service on the canvas and the nodes that contain your search will be highlighted.
+
+== Select your Use Case
```

Comment:
I really like this section! Super clear and a great overview :)

Reply:
What she said!